Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,45 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// A Boolean that is `true` if peer SSL certificates are to be verified.
+    ///
+    /// You can bypass verification by setting this property to `false` (not recommended).
+    public static var verifyPeerSSLCertificate = true
+    /// The URL representing a PEM-encoded certificate authority file to be used for
+    /// peer SSL certificate verification. Setting a certificate authority file takes
+    /// precedence over the value of verifyPeerSSLCertificate.
+    ///
+    /// See:
+    /// - https://curl.haxx.se/docs/sslcerts.html
+    /// - https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL? {
```

Comment:
OK, so it looks like an NDK solution is insurmountable. There simply isn’t enough api to find resources on an app that starts out as Java as far as I can see. The original implementatoin was an enivronment variable avoiding an API but that was not considered secure which is how we arrived here.